testsuite/integration/multinode/pom.xml

```diff
@@ -36,6 +36,14 @@
         <ts.elytron.cli>multinode-setup.cli</ts.elytron.cli>
     </properties>
 
+    <dependencies>
+        <dependency>
+            <groupId>org.wildfly</groupId>
+            <artifactId>wildfly-testsuite-shared</artifactId>
+            <scope>test</scope>
```

Comment:
add `<version>${project.version}</version>`

Reply:
Good tip, thanks ! :)

Reply:
No worries! I'll take a tip against no tip any day :) 

And I also should have say that I was trying to run without the dependency.

Reply:
Hum, I ran into some weird (apparently unrelated) test failures locally. I've still pushed my changes as I want to see if those issues arises in CI. I'll check the result tomorrow.